examples/check-semantics/check_semantics.bal

```diff
@@ -3,7 +3,7 @@ import ballerina/io;
 public function main() returns error? {
     do {
         // If either `foo()` or `bar()` invocations returns an `error`,
-        // the error will be returned from the `main` function and execution
+        // the `error` will be returned from the `main` function and execution
```

Comment:
We emphasize keywords or identifiers. So if we are emphasizing error here we'll have to say something like
```suggestion
        // the `error` value will be returned from the `main` function and execution
```